src/site/content/en/blog/wake-lock/index.md

```diff
@@ -222,6 +222,15 @@ Check out the [Screen Wake Lock demo][demo] and [demo source][demo-source].
 Notice how the screen wake lock is automatically released when you switch tabs
 or apps.
 
+### Screen Wake Locks in the OS task manager
+
+You can use your operating system's task manager to see if an application is preventing
+your computer from sleeping. The video below shows the macOS
+[Activity Monitor](https://support.apple.com/guide/activity-monitor/welcome/mac)
+indicating that Chrome has an active Screen Wake Lock that keeps the system awake.
```

Comment:
```suggestion
indicating that Chrome has an active screen wake lock that keeps the system awake.
```

Reply:
This should be lower case because it refers generically to a class of things and not to the API itself. This is also consistent with capitalization throughout the article.